.github/workflows/debian.yml

```diff
@@ -92,9 +92,20 @@ jobs:
 # I386: actions/checkout@v3,  actions/cache@v3
 #       give error about OCI container [https://github.com/actions/checkout/issues/334]
 #                                      [https://github.com/actions/runner/issues/1011]
-      - name: Checkout
+      - name: Checkout on x64
+        if: arch == 'x64'
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Checkout on ia64
+        if: arch == 'ia32'
```

Comment:
@maxirmx it says "Checkout on ia64" but the condition is `ia32`. Is this correct?

Reply:
It is wrong, thank you, @ronaldtse